database.php

```diff
@@ -0,0 +1,7 @@
+<?php
+$connection = mysqli_connect("localhost", "root", "", "books_db");
```

Comment:
Observations: The code lacks a newline at the end of the file, which is a best practice for POSIX compliance.

Recommendations:
- **Add a Newline**: Ensure there is a newline at the end of the file to adhere to coding standards.